vocabularies/Hierarchy.xml

```diff
@@ -326,7 +348,7 @@ by a PATCH that binds the parent navigation property (for example, `Superordinat
   "method": "patch",
   "url": "$1",
   "body": {
-    "Superordinate": {"@id": "Nodes('B')"}
+    "Superordinate@bind": "Nodes('B')"
```

Comment:
```suggestion
    "Superordinate@odata.bind": "Nodes('B')"
```
Which of our frameworks supports the short form `...@bind`?

Note: CAP Node.js v7.4.2 fails on `...@bind` and silently ignores `...@odata.bind`.